lib/browser/app.js

```diff
@@ -59,6 +59,12 @@ app.controller('AppController', function($q, DriveScannerService, SelectionState
     this.selection.clear();
     this.writer.setProgress(0);
     this.scanner.start(2000);
+
+    this.scanner.scan().then(function (res) {
```

Comment:
I just noticed that the current approach also means that after the first scan, a single drive will not be selected by default anymore. The approach mentioned above should resolve this.